### PR TITLE
Fix/add cvpr meta review revision tests

### PR DESCRIPTION
**Pull request by enrubio**
This adds tests for meta review revision.

### CODE REVIEW

File: tests/test_cvpr_conference_v2.py [hunk 1]
```diff
@@ -740,4 +767,163 @@ def test_secondary_ac_assignment(self, openreview_client, helpers, client):
...
+        helpers.await_queue() 
+
+        # Open meta review revision for final recommendation, allow metareview to be modified
+        due_date = now + datetime.timedelta(days=2)
+        
+        venue.custom_stage = openreview.stages.CustomStage(name='Meta_Review_Revision',
+            reply_to=openreview.stages.CustomStage.ReplyTo.METAREVIEWS,
+            source=openreview.stages.CustomStage.Source.ALL_SUBMISSIONS,
+            reply_type=openreview.stages.CustomStage.ReplyType.REVISION,
+            invitees=[openreview.stages.CustomStage.Participants.AREA_CHAIRS_ASSIGNED],
```
Comment (melisabok) [inline tests/test_cvpr_conference_v2.py:834]: should we include the secondary acs as non invitees here?

Comment (celestemartinez) [reply tests/test_cvpr_conference_v2.py:834]: What I did in my PR (https://github.com/openreview/openreview-py/pull/2017), was automatically add secondary ACs as noninvitees if the invitees include ACs but not secondary ACs. 

If you think it's better, I can just support noninvitees as a customStage param?

Comment (melisabok) [reply tests/test_cvpr_conference_v2.py:834]: ok, that makes sense. @enrubio can you write an assertion where the secondary AC can not submit a metareview revision for the submission they are secondary ACs?

Comment (enrubio) [reply tests/test_cvpr_conference_v2.py:834]: @melisabok @celestemartinez I added a test for this, but I'm not 100% sure it's correct.

File: tests/test_cvpr_conference_v2.py [hunk 2]
```diff
@@ -926,3 +926,22 @@ def test_metareview_revision_stage(self, client, openreview_client, helpers, tes
         assert 'award_candidate' in meta_review.content
 
         assert meta_review.content['metareview']['value'] == 'Revised comment title'
+
+        # Secondary AC can't post meta review revision
+        secondary_ac_client = openreview.api.OpenReviewClient(username='ac1@cvpr.cc', password=helpers.strong_password)
+        secondary_ac_anon_group_id = secondary_ac_client.get_groups(prefix=f'thecvf.com/CVPR/2024/Conference/Submission4/Secondary_Area_Chair_.*', signatory='ac1@cvpr.cc')[0].id
+
+        with pytest.raises(openreview.OpenReviewException, match=r'signatures can only contain the allowed values: thecvf.com/CVPR/2024/Conference/Program_Chairs, thecvf.com/CVPR/2024/Conference/Submission4/Area_Chair_.*'):
```
Comment (melisabok) [inline tests/test_cvpr_conference_v2.py:934]: I was expecting to see an error like "user is not an invitee..." may be we need to merge Celeste's PR first

File: tests/test_cvpr_conference_v2.py [hunk 3]
```diff
@@ -930,4 +957,201 @@ def test_secondary_ac_assignment(self, openreview_client, helpers, client):
...
+            reply_to=openreview.stages.CustomStage.ReplyTo.METAREVIEWS,
+            source=openreview.stages.CustomStage.Source.ALL_SUBMISSIONS,
+            reply_type=openreview.stages.CustomStage.ReplyType.REVISION,
+            invitees=[openreview.stages.CustomStage.Participants.AREA_CHAIRS_ASSIGNED],
+            due_date=due_date,
+            exp_date=due_date + datetime.timedelta(minutes=30),
+            content=meta_review_revision_content)
+
+        venue.create_custom_stage()
+        helpers.await_queue()
```
Comment (melisabok) [inline tests/test_cvpr_conference_v2.py:1036]: this call is waiting for jobs of API 1 to finish. 

Try doing:

```

helpers.await_queue_edit(openreview_client, 'thecvf.com/CVPR/2024/Conference/-/Final_Revision-0-1', count=1)
```

**Review by melisabok — APPROVED** [review]

**Review by celestemartinez — APPROVED** [review]

**Pull request merged**